src/components/review/ReviewsList.js

```diff
@@ -77,6 +78,9 @@ const AlbumReviews = ({ albumId }) => {
           reviewId={reviews[0]?._id}
         />
       )}
+       {user && userHasReviewed && (
+          <DeleteReview reviewId={reviews[0]?._id} refreshReviews={refreshReviews} />
```

Comment:
Hmm, how do you know the current user's review will always be the first one in the `reviews` list?

Reply:
@akosasante just changed this error status on the backend ++ I was asking myself the same thing about passing this  `{reviews[0]?._id} ` as a prop, now added code where use a find method- does it work properly? // also I am thinking maybe I shoul place this 'DELETE' button in another place..